src/java.base/share/classes/java/security/PEM.java

```diff
@@ -90,11 +108,10 @@ public record PEMRecord(String type, String content, byte[] leadingData)
             throw new IllegalArgumentException("PEM syntax labels found.  " +
                 "Only the PEM type identifier is allowed");
         }
-
     }
 
     /**
-     * Creates a {@code PEMRecord} instance with a given {@code type} and
+     * Creates a {@code PEM} instance with a given {@code type} and
      * {@code content} data in String form.  {@code leadingData} is set to null.
```

Comment:
"in String form" is redundant since those are the types of the parameters. Suggest removing those words.